src/restore.c

```diff
@@ -249,10 +249,10 @@ restore_error_t restore_from_directory(const char* dir, Backup* backup, BackupDa
 restore_error_t restore_list_backups(ListBackupsResponse* backups)
 {
     sd_list_t list_subdirs __attribute__((__cleanup__(sd_free_list)));
+    backups->info_count = 0;
```

Comment:
I wonder what tidy was complaining about here?

Reply:
> Uninitialized memory. If the error code is returned you are not allowed to
look at info_count.

But that is okay? One should not inspect the out param at all in case of an error and treat it as undefined.

Reply:
Right, another way to fix it, which probably is cleaner is to initialize the struct in the caller before actually calling `restore_list_backups`.